tests/integration_tests/kafka_messages/run.sh

```diff
@@ -31,10 +31,13 @@ function run_length_limit() {
 	run_cdc_server --workdir $WORK_DIR --binary $CDC_BINARY --loglevel "info"
 
 	TOPIC_NAME="ticdc-kafka-message-test-$RANDOM"
-	SINK_URI="kafka://127.0.0.1:9092/$TOPIC_NAME?protocol=open-protocol&partition-num=4&kafka-version=${KAFKA_VERSION}&max-message-bytes=10485760"
+	# Use a max-message-bytes parameter that is larger than the kafka broker max message bytes.
+	# Test if TiCDC automatically uses the max-message-bytes of the broker.
+	# See: https://github.com/PingCAP-QE/ci/blob/ddde195ebf4364a0028d53405d1194aa37a4d853/jenkins/pipelines/ci/ticdc/cdc_ghpr_kafka_integration_test.groovy#L178
+	SINK_URI="kafka://127.0.0.1:9092/$TOPIC_NAME?protocol=open-protocol&partition-num=4&kafka-version=${KAFKA_VERSION}&max-message-bytes=12582912"
```

Comment:
Is 12582912 the default max-message-bytes of the broker?

Reply:
We set a value larger than the broker configuration. borker's is 11MiB.